src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+import UserService from '../services/userServices';
+import Util from '../utils/response';
+import Helper from '../middlewares/index';
+const { findUserById, updateUser } = UserService;
+const util = new Util();
+
+/** Class representing user controller. */
+class UserController {
+  /**
+  * get user profile
+  * @param {Object} req - server request
+  * @param {Object} res - server response
+  * @returns {Object} - custom response
+  * @description get details of registered user
+  */
+  static async getUserProfile(req, res) {
+    const { id } = req.params;
+    try {
+      const user = await findUserById(id);
+      const token = Helper.generateToken(id);
```

Comment:
Why are you generating token to get a user's profie?

Reply:
I explained this on the PR, the token is generated and use to protect the Patch route... its will be removed when we have a feature that generate token merged to stage

Reply:
OK. Great